WordPress/WordPress.xcodeproj/xcshareddata/xcschemes/Jetpack.xcscheme

```diff
@@ -77,6 +77,56 @@
             ReferencedContainer = "container:WordPress.xcodeproj">
          </BuildableReference>
       </BuildableProductRunnable>
+      <CommandLineArguments>
+         <CommandLineArgument
+            argument = "-debugPinghub"
+            isEnabled = "NO">
+         </CommandLineArgument>
+         <CommandLineArgument
+            argument = "-extra_debug 1"
+            isEnabled = "NO">
+         </CommandLineArgument>
+         <CommandLineArgument
+            argument = "-force-crash-logging 1"
+            isEnabled = "NO">
+         </CommandLineArgument>
+         <CommandLineArgument
+            argument = "-com.apple.CoreData.ConcurrencyDebug 1"
+            isEnabled = "NO">
+         </CommandLineArgument>
+         <CommandLineArgument
+            argument = "-wpcom-api-base-url http://localhost:8282/"
+            isEnabled = "NO">
+         </CommandLineArgument>
+         <CommandLineArgument
+            argument = "-debugJetpackConnectionFlow"
+            isEnabled = "NO">
+         </CommandLineArgument>
+      </CommandLineArguments>
```

Comment:
`git grep` for there is code that checks for each of them, so I guess it's useful to have them around.

Funny enough (or not) there are different ways in which they are read. E.g.:

https://github.com/wordpress-mobile/WordPress-iOS/blob/3cd51cf3cf39d3d5a9c4fd02301ed10f7ccbf1aa/WordPress/Classes/ViewRelated/Jetpack/Install/Webview/JetpackConnectionWebViewController.swift#L345

https://github.com/wordpress-mobile/WordPress-iOS/blob/3cd51cf3cf39d3d5a9c4fd02301ed10f7ccbf1aa/WordPress/Classes/Models/UserSettings.swift#L12-L19

https://github.com/wordpress-mobile/WordPress-iOS/blob/3cd51cf3cf39d3d5a9c4fd02301ed10f7ccbf1aa/WordPress/Classes/Utility/PingHubManager.swift#L312-L314